cli/report/report.py

```diff
@@ -532,3 +545,25 @@ def _get_open_bugs(
             return open_bugs
         else:
             return None
+
+    def _get_success_issue_description(
```

Comment:
Do we need another function for this?
`_get_issue_description` can get `success` arg and build the issue to match it